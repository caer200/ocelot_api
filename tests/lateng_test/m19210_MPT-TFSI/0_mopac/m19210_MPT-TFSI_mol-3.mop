CHARGES

 
H	13.242335 +1 1.361714 +1 11.531559 +1
C	13.962620 +1 1.310123 +1 12.148964 +1
C	14.006291 +1 0.256232 +1 13.079860 +1
C	14.944428 +1 2.263750 +1 12.121749 +1
C	15.060940 +1 0.274056 +1 14.029984 +1
N	13.059696 +1 -0.758891 +1 13.043710 +1
H	14.902518 +1 2.954149 +1 11.470848 +1
C	16.003392 +1 2.243697 +1 13.025616 +1
C	16.051378 +1 1.265561 +1 13.979985 +1
S	15.252192 +1 -0.894916 +1 15.269777 +1
C	12.951987 +1 -1.770225 +1 13.993034 +1
C	12.161950 +1 -0.811028 +1 11.880148 +1
H	16.685444 +1 2.904005 +1 12.979936 +1
H	16.760081 +1 1.256994 +1 14.612664 +1
C	13.883220 +1 -1.912824 +1 15.043202 +1
C	11.894231 +1 -2.691545 +1 13.937122 +1
H	11.937781 +1 -1.743834 +1 11.681691 +1
H	11.341412 +1 -0.312647 +1 12.078087 +1
H	12.608233 +1 -0.410037 +1 11.105035 +1
C	13.761702 +1 -2.954461 +1 15.963927 +1
H	11.241540 +1 -2.615244 +1 13.251190 +1
C	11.790837 +1 -3.700874 +1 14.859118 +1
H	14.404054 +1 -3.046314 +1 16.657822 +1
C	12.725867 +1 -3.845701 +1 15.877109 +1
H	11.066168 +1 -4.312410 +1 14.801336 +1
H	12.647502 +1 -4.555072 +1 16.504578 +1
