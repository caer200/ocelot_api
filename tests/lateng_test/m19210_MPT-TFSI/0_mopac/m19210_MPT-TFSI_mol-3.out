 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 21:33:10 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H         13.24233500  *   1.36171400  *  11.53155900  *
     2       C         13.96262000  *   1.31012300  *  12.14896400  *
     3       C         14.00629100  *   0.25623200  *  13.07986000  *
     4       C         14.94442800  *   2.26375000  *  12.12174900  *
     5       C         15.06094000  *   0.27405600  *  14.02998400  *
     6       N         13.05969600  *  -0.75889100  *  13.04371000  *
     7       H         14.90251800  *   2.95414900  *  11.47084800  *
     8       C         16.00339200  *   2.24369700  *  13.02561600  *
     9       C         16.05137800  *   1.26556100  *  13.97998500  *
    10       S         15.25219200  *  -0.89491600  *  15.26977700  *
    11       C         12.95198700  *  -1.77022500  *  13.99303400  *
    12       C         12.16195000  *  -0.81102800  *  11.88014800  *
    13       H         16.68544400  *   2.90400500  *  12.97993600  *
    14       H         16.76008100  *   1.25699400  *  14.61266400  *
    15       C         13.88322000  *  -1.91282400  *  15.04320200  *
    16       C         11.89423100  *  -2.69154500  *  13.93712200  *
    17       H         11.93778100  *  -1.74383400  *  11.68169100  *
    18       H         11.34141200  *  -0.31264700  *  12.07808700  *
    19       H         12.60823300  *  -0.41003700  *  11.10503500  *
    20       C         13.76170200  *  -2.95446100  *  15.96392700  *
    21       H         11.24154000  *  -2.61524400  *  13.25119000  *
    22       C         11.79083700  *  -3.70087400  *  14.85911800  *
    23       H         14.40405400  *  -3.04631400  *  16.65782200  *
    24       C         12.72586700  *  -3.84570100  *  15.87710900  *
    25       H         11.06616800  *  -4.31241000  *  14.80133600  *
    26       H         12.64750200  *  -4.55507200  *  16.50457800  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H         13.2423      1.3617     11.5316
     2         C         13.9626      1.3101     12.1490
     3         C         14.0063      0.2562     13.0799
     4         C         14.9444      2.2637     12.1217
     5         C         15.0609      0.2741     14.0300
     6         N         13.0597     -0.7589     13.0437
     7         H         14.9025      2.9541     11.4708
     8         C         16.0034      2.2437     13.0256
     9         C         16.0514      1.2656     13.9800
    10         S         15.2522     -0.8949     15.2698
    11         C         12.9520     -1.7702     13.9930
    12         C         12.1619     -0.8110     11.8801
    13         H         16.6854      2.9040     12.9799
    14         H         16.7601      1.2570     14.6127
    15         C         13.8832     -1.9128     15.0432
    16         C         11.8942     -2.6915     13.9371
    17         H         11.9378     -1.7438     11.6817
    18         H         11.3414     -0.3126     12.0781
    19         H         12.6082     -0.4100     11.1050
    20         C         13.7617     -2.9545     15.9639
    21         H         11.2415     -2.6152     13.2512
    22         C         11.7908     -3.7009     14.8591
    23         H         14.4041     -3.0463     16.6578
    24         C         12.7259     -3.8457     15.8771
    25         H         11.0662     -4.3124     14.8013
    26         H         12.6475     -4.5551     16.5046

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C13 H11 N S  =    26 atoms



      MOLECULAR POINT GROUP   :   Cs  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
